Gopkg.lock

```diff
@@ -1077,108 +888,29 @@
     "util/buffer",
     "util/cert",
     "util/flowcontrol",
-    "util/integer",
+    "util/integer"
   ]
-  pruneopts = "NUT"
   revision = "78700dec6369ba22221b72770783300f143df150"
   version = "v6.0.0"
 
 [[projects]]
   branch = "master"
-  digest = "1:e2e5161f8bd8965992891a1b600546080caea658af1a246c9e355b20b303a844"
   name = "k8s.io/kube-openapi"
   packages = [
     "pkg/common",
-    "pkg/util/proto",
+    "pkg/util/proto"
   ]
-  pruneopts = "NUT"
   revision = "275e2ce91dec4c05a4094a7b1daee5560b555ac9"
 
 [[projects]]
-  digest = "1:6ce53e143b429d0e14d7dc0b4ebd1d6f15427e125aa5c3ef4ccf5f34fa72e359"
   name = "k8s.io/kubernetes"
   packages = ["pkg/kubelet/apis/stats/v1alpha1"]
-  pruneopts = "NUT"
   revision = "f01a2bf98249a4db383560443a59bed0c13575df"
   version = "v1.9.5"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  input-imports = [
```

Comment:
There is some thrash from different versions of go dep.  Did you use the version in the dev image?

Reply:
I did